.github/workflows/publish-release.yml

```diff
@@ -76,7 +99,9 @@ jobs:
       - uses: actions/cache@v3
         id: restore-build
         with:
-          path: ./dist
+          path: |
+            ./dist
+            ./docs
```

Comment:
Are the docs needed here? We intentionally kept the cache minimal for the publishing step.

Reply:
Since we can remove the staging deployment, this is definitely not needed anymore. I wasn't sure if the cache would be restored if the `path` was different, but it's irrelevant now.